datahub/core/realtime_messaging.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from django.conf import settings
+from slack import WebClient
+from slack.errors import SlackClientError  # the base error class
+
+logger = logging.getLogger(__name__)
+
+
+class RealtimeMessaging():
+    """
+    Helper class to enable transmission of messages to realtime services.
+
+    In this case, Slack.
+    """
+
+    _config = None
+
+    def __init__(self):
+        """
+        Set up configuration for Slack client.
+        """
+        config = dict()
+        for setting in 'SLACK_API_TOKEN', 'SLACK_MESSAGE_CHANNEL':
+            setting_value = getattr(settings, setting, None)
+            if setting_value:
+                config[setting] = setting_value
+            else:
+                logger.info(
+                    f'Missing setting `{setting}`. Skipping setup.')
+        if len(config.keys()) == 2:
+            self._config = config
+
+    def send_realtime_message(self, message_text):
+        """
+        Send a message to the realtime messaging system.
+
+        Do not attempt if the messaging config was not set up properly.
+        """
+        if self._config is None:
+            logger.info('Setup was incomplete. Message not attempted.')
+            return
+        client = WebClient(
+            timeout=settings.SLACK_TIMEOUT_SECONDS,
+            token=self._config['SLACK_API_TOKEN'])
+        try:
+            client.chat_postMessage(
+                channel=self._config['SLACK_MESSAGE_CHANNEL'],
+                text=message_text)
+        except SlackClientError:
+            logger.error('Slack post failed.', exc_info=True)
```

Comment:
For info, this can be simplified to:
```suggestion
            logger.exception('Slack post failed.')
```

Reply:
Thanks, changing.